docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -283,16 +283,13 @@ Database resource group manager
 
 This example is for a MySQL database.
 
-As stated above, the environment ``'test'`` in the example should match ``node.environment`` property from
-:ref:`node_properties`.
-
 .. code-block:: sql
 
     -- global properties
     INSERT INTO resource_groups_global_properties (name, value) VALUES ('cpu_quota_period', '1h');
 
-    -- Every row in resource_groups table indicates a resource group. The parent-child relationship
-    -- is indicated by the ID in 'parent' column.
+    -- Every row in resource_groups table indicates a resource group. The enviroment name is 'test'.
```

Comment:
```suggestion
    -- Every row in resource_groups table indicates a resource group. The enviroment name is 'test_environment'.
```

And let's change queries to reference that. That should make it even clearer since we use the environment value in lots of queries.